### PR TITLE
Merge Wisp to Misk

### PR DESCRIPTION
**Pull request by rainecp**
This merges [wisp repo](https://github.com/cashapp/wisp) as a new module in misk

### CODE REVIEW

**Review by r3mariano — APPROVED** [review]

File: wisp/build.gradle.kts [hunk 1]
```diff
@@ -1,178 +1,12 @@
...
-    }
-  }
-
-  repositories {
-    mavenCentral()
-    maven(url = "https://s3-us-west-2.amazonaws.com/dynamodb-local/release")
-  }
 
-  if (!path.startsWith(":wisp-bom")) {
+  if (!path.startsWith(":wisp:wisp-bom")) {
```
Comment (adrw) [inline wisp/build.gradle.kts:6]: Is this file necessary? I've usually seen the subdir not have a build file.

**Review by adrw — APPROVED** [review]
LG

File: wisp/build.gradle.kts (hunk 1 above)
Comment (rainecp) [reply wisp/build.gradle.kts:3]: For now, yes, because unlike Misk which adds the plugins in each submodule's `build.gradle`, wisp defines them in top level build.gradle. This minimizes the changes for this merge and makes it easier to debug and test the migration changes for wisp modules as well as look at previous changes on the same file. But in the future, we can cleanup Misk and just apply common plugins in the main build.file.
